### ISSUE TITLE
Add "jump to next parent comment" 

### ISSUE THREAD

**Issue by sageDieu** [thunder-app/thunder#313]
**Is your feature request related to a problem? Please describe.**
When browsing a large comment section it can be easy to lose your place or get deep into a nested chain and want a quick way to jump to the next parent comment.

**Describe the solution you'd like**
Apollo, Reddit Sync, and many other great reddit apps offered basically a floating button that appears at all times over the comments when viewing a post. I got used to this and the UX works well. It's like the same as Thunder's existing reply button but with an arrow icon.

Basically a short tap on this button would jump scroll the user to where the next parent comment is at the top of their screen, and a long press would do the same in reverse to jump to the previous parent.

For a quick example check out the HackerNews app HACK: https://apps.apple.com/us/app/hack-for-hacker-news-reader/id1464477788

This function is enabled by default so just view any post in that app and you'll see an arrow button in the middle to tap. (Personally I'd rather it be on the right side than the middle)

**Comment by micahmo** [comment]
Related to #75.
